cmd/evmosd/testnet.go

```diff
@@ -73,6 +75,8 @@ type initArgs struct {
 	numValidators     int
 	outputDir         string
 	startingIPAddress string
+	baseFee           sdkmath.Int
+	minGasPrice       sdkmath.Int
```

Comment:
When subsequently passing this value to feemarket as params, I converted Int to Dec.

Dec is actually because in the CheckTx stage, this accuracy is needed for calculation. In pass parameters as  scenarios, I actually think Int is enough,  Of course, I have changed minGasPrice to Dec.